### ISSUE TITLE
@capacitor/push-notifications bring app to fron on message receive

### ISSUE THREAD

**Issue by donmb1** [ionic-team/capacitor-plugins#738]
### Plugin
@capacitor/push-notifications

### Description
With FCM and native code it is somehow possible to bring an app to front when it receives a message. This makes sense to include it into @capacitor/push-notifications plugin as a method which can get fired on `pushNotificationReceived`

### Platform(s)
Android

### Preferred Solution

A method which brings the App to foreground after a push notification was received.

### Additional Context
In our use case the app is in background and the user interacts with another app. Now an alarm comes in and the user has to check the app to see which alarm they received. The app should come to foreground as its an important message which a standard push notification is not suitable for.

**Comment by jcesarmobile** [comment]
> With FCM and native code it is somehow possible to bring an app to front when it receives a message.

Can you provide more information about that?

**Comment by donmb1** [comment]
Yes, there are several threads on StackOverflow and other communities which indicate that this is possible

Examples:
https://stackoverflow.com/questions/65476441/android-bring-app-to-foreground-on-firebase-notification
https://www.py4u.net/discuss/700667

Unfortunately I am not a native App dev (which is why I'm using capacitor ✌️ ) and this is all Chinese to me.

**Comment by donmb1** [comment]
Alternatively this feature also makes sense too add to `[https://capacitorjs.com/docs/apis/app](@capacitor/app)`

**Comment by eric-horodyski** [comment]
This thread has been inactive for some time, and we’re cleaning up older items to keep the backlog focused and manageable.

Details on the effort:
[https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant today, feel free to open a new issue with current information and we’ll review it.
